<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NoteClass</class>
 <widget class="QMainWindow" name="NoteClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Note</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="ImageCalendar" name="imageCalendar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>140</width>
      <height>158</height>
     </rect>
    </property>
   </widget>
   <widget class="LedNumber" name="ledNumberL">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>20</y>
      <width>111</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="LedNumber" name="ledNumberR">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>20</y>
      <width>111</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditNote">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>421</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_NextSee">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>30</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>距离见面还有：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Note">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>备忘：</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>120</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Together</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>我们在一起：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_Together">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>61</y>
       <width>81</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButtonHide">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>190</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>隐藏程序</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>LedNumber</class>
   <extends>QWidget</extends>
   <header>lednumber.h</header>
  </customwidget>
  <customwidget>
   <class>ImageCalendar</class>
   <extends>QWidget</extends>
   <header>imagecalendar.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="note.qrc"/>
 </resources>
 <connections/>
</ui>
